src/InPluginUpsells/resources/js/AdditionalAddonCard.js

```diff
@@ -0,0 +1,15 @@
+import {Button} from './Button';
+import {Card} from './Card';
+
+import styles from './AdditionalAddonCard.module.css';
+
+export const AdditionalAddonCard = ({name, description, image, actionLink, actionText}) => (
+    <Card as="article" className={styles.card}>
+        <img className={styles.image} src={image} alt="" />
```

Comment:
Since this [image is decorative](https://www.w3.org/WAI/tutorials/images/decorative/), I’m adding the attribute and setting it as empty so that screen readers don’t read out the URL (i.e. what might happen if no `alt` attribute was set) or repeat information that’s already been communicated by the surrounding context (i.e. the name in the `h3` tag or description of the addon in the `p` tag which follow the image).